src/elvis_webhook.erl

```diff
@@ -22,13 +22,11 @@ event(Cred, Request) -> egithub_webhook:event(?MODULE, Cred, Request).
 -spec handle_pull_request(
   egithub:credentials(), egithub_webhook:req_data(),
   [egithub_webhook:file()]) ->
-  {ok, [egithub_webhook:message()]} | {error, term()}.
+  {ok, egithub_webhook:review()} | {error, term()}.
 handle_pull_request(Cred, Data, GithubFiles) ->
     #{<<"repository">> := Repository} = Data,
-    BranchName = maps_get([ <<"pull_request">>,
-                            <<"head">>,
-                            <<"ref">>],
-                            Data, <<"master">>),
+    BranchName =
+      maps_get([<<"pull_request">>, <<"head">>, <<"ref">>], Data, <<"master">>),
```

Comment:
Are you sure <<"master">> should be hardcoded here?
There are PRs that don't go against `master`.